runtime/tutor

```diff
@@ -1007,7 +1007,10 @@ lines.
  --> How much would would a wouldchuck chuck
  --> if a wouldchuck could chuck would?
 
-
+ NOTE: If typing ) or ( appears to do nothing, try using a 
+ different color theme. These themes work well here:
+ - dark_plus
+ - emacs
```

Comment:
```suggestion
 NOTE: If typing ) or ( appears to do nothing, try using the
 default color theme.
```

Reply:
ofcourse this will come after the default theme is fixed...

Reply:
> aahw, nice. fair point. i was thinking about its future use, but I wasn't able to see how it will work together, what will be the story behind.
>
>say a user installs helix, and changes the theme in due course, but then after some time uses the helix tutor.
>
>so, for those cases too, i'd like to point users to default theme. 'cz :
>
>    hey, the default theme should work out of the box for all the crucial things.
>    and we shouldn't refer to different themes on a per-case basis.
>
> but ofcourse, that requires fixing the default theme first. that's what that other pr is for
>
> ofcourse this will come after the default theme is fixed...

That's an interesting approach.  I'm not sure it has quantifiable advantages because this PR
- resolves a current problem without depending on a future PR fixing any theme
- mitigates a future problem if the user chooses a buggy theme we don't know about
- doesn't prevent default theme from being suggested once it is fixed (the PR fixing the default theme can include a commit to suggest using it inside the tutorial)

Why not merge this PR as-is first, then once the default theme is fixed, open a new PR to include it as a recommended workaround?  This way would be less fragile because it doesn't introduce external dependencies.

If the next release doesn't include a fix for the default theme (or other problematic themes), then this PR would still be able to help new users and reviewers trying Helix tutorial.

Reply:
> Why not merge this PR as-is first, then once the default theme is fixed, open a new PR to include it as a recommended workaround?

i don't have any concern with the order of it, the suggestion i gave is for how it should end eventually in my views.

To reiterate, i don't have problem with what you said.